rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+
+    # Update associations
+    book.add_rental(self)
+    person.add_rental(self)
```

Comment:
- [ ] Hello! 😊 How about adding both the `book` and `person` to the rentals array, which is defined in the constructors of their respective classes? 📚👤 This way, you'll have a comprehensive record of `rentals` right from the start, making your project's data structure even more informative and user-friendly! If you'd like assistance with implementing this, feel free to ask. Happy coding! 🚀📖
```suggestion
      @book = book
    # *The rental is adding itself to the @book achieving the 'belongs-to' association
    book.rentals << self

    @person = person
    # *The rental is adding itself to the @person the 'belongs-to' association
    person.rentals << self
```

Reply:
code updated accordingly